layouts/resources/list.html

```diff
@@ -0,0 +1,51 @@
+{{ define "hero" }}
+    <header class="header-hero-dark py-0 pt-16 px-4">
+        <div class="container mx-auto text-white md:flex justify-around items-center">
+            <div class="md:w-1/2">
+                <h1 class="text-white text-center">Resources</h1>
+            </div>
+            <div class="md:w-1/2 md:ml-8">
+                <div class="text-center">
+                    <img class="inline-block w-56" src="/icons/icon-audience-devops.svg">
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <img class="w-full h-20" src="/images/waves.svg">
+{{ end }}
+
+{{ define "main" }}
+    <!-- Grab the webinar and event page data. -->
+    {{ $webinars := (where $.Pages "Type" "webinars") }}
+    {{ $events := (where $.Site.Pages "Type" "events") }}
+
+    <!-- Remove any webinar/events that are marked as unlisted. -->
+    {{ $listed_webinars := (where $webinars ".Params.unlisted" false) }}
+    {{ $listed_events := (where $events ".Params.unlisted" "!=" true )}}
+
+    <!-- Convert the webinar data to JSON -->
+    {{ $eventListWebinars := slice }}
+    {{ range $listed_webinars }}
+        {{ $eventListWebinars = $eventListWebinars | append (.Params | jsonify)}}
+    {{ end }}
+
+    <!-- Conver the event data to JSON -->
+    {{ $eventListEvents := slice }}
+    {{ range $listed_events }}
+        {{ if ne .Params.title "Upcoming Events" }}
+            {{ $eventListEvents = $eventListEvents | append (.Params | jsonify)}}
+        {{ end }}
+    {{ end }}
+
+    <div class="pulumi-event-list-container mt-5">
+        <pulumi-event-list
```

Comment:
The thing about this approach that we need to be super careful about is that because these items are all rendered with JS now, we're at risk of adversely affecting the crawlability of this page. Today, for example, /webinars renders a list of links in the source of the page; we do a bit of fiddling with the ordering of things, but the source markup is baked into what we deliver to the requestor. But here, we no longer have that. I suspect we'll want to make sure the "view source" content we end up delivering somehow exposes `<a href="/link-to-thing">thing name</a>` in a bot-friendly way, unless we're absolutely sure the bots will execute our JS. That might be the case, but it's something I'd definitely want to doublecheck before proceeding with this as written.

Reply:
I realized this morning that in this light, it's also worth asking: since the relevant thing here is just to be able to filter (which really means hide, and which can easily be done with CSS), why do the items themselves need to be components at all? If you had just the filtering controls as a component, and that component were able to set a prop or a class on itself, wouldn't that be enough? I think it would. 🤔 

```
<pulumi-event-filter selected="all"></filter>
<ol class="pulumi-event-items">
    <li class="webinar"></li>
    <li class="event"></li>
    ...
</ol>
```

```
pulumi-event-filter {

    // Adjacent sibling selects the list itself; all items are hidden/invisible by default.
    + .pulumi-event-items {
        li {
            @apply hidden;
        }
    }

    // Items matching what's selected are visible.
    &[selected="webinar"] + .pulumi-event-items li.webinar,
    &[selected="event"] + .pulumi-event-items li.event,
    ... {
        display: block;
    }
}
```